README.md

```diff
@@ -1,12 +1,24 @@
 # 18F Compliance Toolkit
-Towards a more secure (and automated) future.
+
+_Towards a more secure (and automated) future._
 
 ## Project Goals
 
+* Get ATOs completed in fewer hours
+* Reduce the burden on the Information Security team leading up to new projects launching
+* Assure greater security over 18F's projects in the long term
+* Establish best practices for the security and compliance of our projects
+
```

Comment:
Should we add an item along the lines of:

Inform project teams of the necessary steps to acquire an ATO.